includes/definitions/discovery/fiberhome-switch.yaml

```diff
@@ -0,0 +1,60 @@
+modules:
+  processors:
+        data:
+            -
+                 oid: WRI-CPU-MIB::cpuTable
```

Comment:
All the MIBs you've specified need to be added at the top of this file like:

`mib: WRI-CPU-MIB:WRI-TEMPERATURE-MIB:etc....` and removed from the `oid` lines.

Reply:
noted and done.